pym/bob/cmds/build.py

```diff
@@ -1085,6 +1087,7 @@ def doProject(argv, bobRoot):
     if args.execute_prebuild:
         devArgs = extra.copy()
         if args.resume: devArgs.append('--resume')
+        if args.execute_buildonly: devArgs.append('-b')
```

Comment:
Thanks :) ... Just saw this while reviewing your PR. Looks like nobody used this before...